src/components/organisms/ChatSidebar/ChatSidebar.tsx

```diff
@@ -19,17 +25,33 @@ export const _ChatSidebar: React.FC = () => {
       ? chatSettings.recipient
       : undefined;
 
-  const venueTabId = "chat-sidebar-tab-venue";
-  const privateTabId = "chat-sidebar-tab-private";
+  if (isLoading) {
+    // No need to show anything in this case
+    // loading should be quick and from cache
+    return null;
+  } else if (!userId) {
+    // But in this case, chat has been used in a in a place with no user logged in
+    captureError(
+      new SparkleAssertError({
+        message:
+          "ChatSidebar, as a descendant of LoginRestricted, should have userId",
+        where: "ChatSidebar",
+        args: { isLoading, userId, chatSettings },
+      })
+    );
+    return null;
+  }
 
   return (
     <div
       role="dialog"
-      aria-labelledby={isVenueChat ? venueTabId : privateTabId}
+      aria-labelledby={isVenueChat ? CHAT_TAB_SPACE_ID : CHAT_TAB_PRIVATE_ID}
       className={styles.chatSidebar}
     >
       {isVenueChat && <VenueChat />}
-      {isPrivateChat && <PrivateChats recipient={recipient} />}
+      {isPrivateChat && userId && (
```

Comment:
[Question] Can this chat type be something other than `venue` or `private`?
If not we might not need the `isVenueChat` and use a ternary here rendering `PrivateChats` if `isPrivateChat && userId` are `true` and `VenueChat` if it's `false`